application/src/views/Project.vue

```diff
@@ -0,0 +1,308 @@
+<template>
+  <div class="main-project">
+    <div class="grid-item section__2">
+        <img class="author-pic" :src="project.owner.image" :title="project.owner.name" :alt="project.owner.name"/>
+        <p>{{project.owner.name}}</p>
+        <p> Launched {{project.created_on}}</p>
+    </div>
+    <div class="grid-item section__4">
+      <h1>{{project.name}}</h1>
+      <h2>{{project.introduction}}</h2>
+    </div>
+    <div class="grid-item section__3">
+      <img class="project-pic" :src="project.image">
+      <p class="details">Email@Email.com</p>
+      <p class="details"><font-awesome-icon class="blue-title" icon="globe-americas" /> Helsinki, Fi</p>
+      <p class="details tag">{{project.tag}}</p>
+    </div>
+    <div class="grid-item section__3">
+      <div class="project-status">
+        <h1 class="blue-title">4</h1>
```

Comment:
```suggestion
        <h1 class="blue-title">{{project.members.length}}</h1>
```

The db does not make a distinction between original and new team members. So I would suggest we replace the 'Original team members' and 'Amazing new members' for 'Team Members'.

Reply:
Project view still needs refactoring, I switched to the project creation view so that I can come back on it afterwards, but I can make this change already